BraveWallet/WalletConstants.swift

```diff
@@ -47,5 +47,5 @@ struct WalletConstants {
 }
 
 struct WalletDebugFlags {
-  static var isSolanaEnabled: Bool = false
+  static var isSolanaEnabled: Bool = true
```

Comment:
We could remove this flag altogether maybe, sometime in the future

Reply:
I agree, I would suggest dropping this flag entirely (and just include `.sol` in `supportedCoinTypes` array) would be good in this PR.

Reply:
removed this flag in [5a674dd](https://github.com/brave/brave-ios/pull/5719/commits/5a674ddab0e3e5b440cc259786a44e874b8fea3a)
re-requesting the team to review.